### PR TITLE
Emergency flow page

### LINKED ISSUE #41: Adding Emergency Flow
Implement this page for adding/editing an injury. Exclude the"Preview" and "Save Draft" buttons for now. Contact me if you have any questions about the backend side of things. Also, no need to do the checkboxes with the different emergency types for now, that'll be handled later.

image

### PR DESCRIPTION
**Pull request by r800360**
## Tracking Info

Resolves #41 

## Changes

Designed the Emergency flow page and added functionality to enter into text fields and publish to MongoDB database

## Testing

To test, try entering into the text fields and click publish to send to MongoDB database.

A screenshot of the page is as follows:
EmergencyFlowPage

### CODE REVIEW

**Review by Anthonyp0329 — CHANGES_REQUESTED** [review]
Looking awesome! Just a few small things: when the user doesn't put anythgin under one of the optional fields, make sure to not include that field in the object that gets published into the database. Also - experiencing some small styling issues with the textinputs overlapping the navbar: 
image

**Comment by Anthonyp0329** [comment]
also just remembered, make sure to add a subtitle field at the top. This is the part that is displayed on the card for that emergency

File: admin-portal-frontend/emergencies.ts
Comment (Anthonyp0329) [inline admin-portal-frontend/emergencies.ts:1]: Move this, as well as the requests.ts file into the src/app folder in a folder called api or something of the sort

File: package.json
Comment (Anthonyp0329) [inline package.json:1]: remove this

File: admin-portal-frontend/emergencies.ts [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+/**
+ * The implementations of these API client functions are provided as part of the
+ * MVP. You can use them as a guide for writing the other client functions.
+ */
+export async function createEmergency(
+  emergency: CreateEmergencyRequest,
+): Promise<APIResult<Emergency>> {
+  try {
+    const response = await post("/api/emergencies", emergency);
```
Comment (Anthonyp0329) [inline admin-portal-frontend/emergencies.ts]: make sure to have these routes depend on a .env file for local development purposes

**Review by Anthonyp0329 — CHANGES_REQUESTED** [review]
Almost ready to go! Just address the changes I noted, and also make sure to have some sort of error message notifying the user when a required field isn't filled out when they try to submit the form.

**Comment by r800360** [comment]
Issues Addressed/Changelog:
-- when the user doesn't put anything under an optional field (any field not title or subtitle), field is not included in the object that gets published into the database

-- fixed styling issues with the textinputs overlapping the navbar

-- added subtitle field to the form

-- added error message notifying the user when a required field (title or subtitle) isn't filled out when they try to submit the form

-- moved emergencies.ts and requests.ts into src/app/api

-- remove head package.json (separate from other package.json files)

-- Created dependency between src/api/emergencies.ts (createEmergency(), getEmergency(), getAllEmergencies(), updateEmergency()) on local.env for the admin-portal-frontend

Thanks! Let me know if we missed anything.

**Pull request merged**